include/sys/vdev_impl.h

```diff
@@ -187,6 +199,10 @@ struct vdev {
 	boolean_t	vdev_ishole;	/* is a hole in the namespace	*/
 	kmutex_t	vdev_queue_lock; /* protects vdev_queue_depth	*/
 	uint64_t	vdev_top_zap;
+	uint64_t	vdev_alloc_bias_birth; /* allocation bias start	*/
+	uint64_t	vdev_last_io;	  /* lbolt of last non-scan I/O */
+	nvlist_t	*vdev_cfg;    /* additional dRAID configuration */
```

Comment:
Honestly I think this is a bad split from another merged tree. There are items missing in `cmd/zpool/zpool_vdev.c` which makes it impossible to specify `"metadata"` as a keyword to creating a pool/vdev.